draft-ietf-cose-hash-envelope.md

```diff
@@ -115,7 +115,7 @@ Hash_Envelope = #6.18(Hash_Envelope_as_COSE_Sign1)
 
 - Label `1` (alg) Cryptographic algorithm to use
 - Label `258` (payload hash alg) MUST be present in the protected header and MUST NOT be present in the unprotected header.
-- Label `259` (content type of the preimage of the payload) MAY be present in the protected header or unprotected header.
+- Label `259` (content type of the preimage of the payload) MUST be present in the protected header and MUST NOT be present in the unprotected header.
```

Comment:
```suggestion
- Label `259` (content type of the preimage of the payload) MAY be present in the protected header and MUST NOT be present in the unprotected header.
```

Reply:
Your proposed change makes this parameter required.

Your intention is to make it optional, but only allowed in the protected header.

Reply:
the CDDL shows required, so which do we want to be consistent with?

https://cose-wg.github.io/draft-ietf-cose-hash-envelope/draft-ietf-cose-hash-envelope.html#section-4

Reply:
We don't want anything other than hash algorithm to be required.

Reply:
I fixed the CDDL above